### ISSUE TITLE
ansible-playbook 01_general-server-configuration.yml fails at "TASK [chrisvanmeer.hashicorp : Include right OS task file.]"

### ISSUE THREAD

**Issue by JalfResi** [chrisvanmeer/at-hashi-demo#5]
fatal: [server1]: FAILED! =>
  reason: Could not find or access '/home/bendavies/at-hashi-demo/setup-hashicorp-Ubuntu.yml' on the Ansible Controller.
fatal: [server2]: FAILED! =>
  reason: Could not find or access '/home/bendavies/at-hashi-demo/setup-hashicorp-Ubuntu.yml' on the Ansible Controller.
fatal: [server3]: FAILED! =>
  reason: Could not find or access '/home/bendavies/at-hashi-demo/setup-hashicorp-Ubuntu.yml' on the Ansible Controller.
fatal: [client1]: FAILED! =>
  reason: Could not find or access '/home/bendavies/at-hashi-demo/setup-hashicorp-Ubuntu.yml' on the Ansible Controller.
fatal: [client2]: FAILED! =>
  reason: Could not find or access '/home/bendavies/at-hashi-demo/setup-hashicorp-Ubuntu.yml' on the Ansible Controller.
fatal: [client3]: FAILED! =>
  reason: Could not find or access '/home/bendavies/at-hashi-demo/setup-hashicorp-Ubuntu.yml' on the Ansible Controller.
fatal: [client4]: FAILED! =>
  reason: Could not find or access '/home/bendavies/at-hashi-demo/setup-hashicorp-Ubuntu.yml' on the Ansible Controller.

Later on, this causes "ansible-playbook 03_consul-deployment.yml" to fail with "TASK [HashiCorp - Consul : Ensure data directory is present.]"

**Comment by chrisvanmeer** [comment]
Did you forget to install the role or edit any of the code perhaps?  
I can't see why this is happening.

**Comment by JalfResi** [comment]
Nope, installed the roles and didnt edit any of the code. Bizzarly, when I add a symbolic link:

ln -s /home/bendavies/at-hashicorp-demo/set-hashicorp-Ubuntu.yml /home/bendavies/.ansible/roles/chrisvanmeer.hashicorp/tasks/setup-hashicorp-Debian.yml 

it appears to work.

Not particulary familiar with Ansible yet - but why would the "Include right OS task file" task look in the project root for the role?

**Comment by chrisvanmeer** [comment]
Sounds like you have a `roles_path` defined in an Ansible config somewhere with `./` as value that overwrites.  

If I use the role, I see a task

```
TASK [chrisvanmeer.hashicorp : Include right OS task file.] ***************************************************************************************
included: /Users/chris/.ansible/roles/chrisvanmeer.hashicorp/tasks/setup-hashicorp-Ubuntu.yml for acme-vault1
```

**Comment by chrisvanmeer** [comment]
Closing since this has become stale.
